### DISCUSSION TITLE
Proxified an application behind an ssh jump host

### DISCUSSION THREAD

**Discussion by crile** [gravitational/teleport#6289] [Q&A] [answered]
I want to add an application in teleport.
This applications is reachable only with an SSH tunnel from the server 1 to the server 2.
I can't install teleport on a remote host

```
|----------|      |--------------|                  |---------------|
| Teleport |------| SSH server 1 |---[ssh tunnel]---| SSH server 2  |
|          |      |              |                  | application A |
|----------|      |--------------|                  |---------------|
```

Can I configure Teleport to acces this application ?

**Comment by webvictim** [comment, marked as answer]
This isn't a supported use case directly via Teleport at the moment. However, if you could configure something which will keep a persistent SSH tunnel running from `server1` to `server2` (say on local port 3000) you could theoretically configure the Teleport `app_service` on `server1` and add an app running on `http://localhost:3000`. When someone attempts to access the app, it should send the traffic to port 3000 which would then be sent over the SSH tunnel.

On `server1`: `ssh -L 3000:127.0.0.1:<remote app port> server2`

**Comment by crile** [comment]
ok, thanks
